packages/modules/delete-imodel/src/components/DeleteIModel.tsx

```diff
@@ -107,7 +115,7 @@ export function DeleteIModel(props: DeleteIModelProps) {
   }, [onClose]);
 
   return (
-    <ThemeProvider theme="inherit">
+    <>
```

Comment:
oh wow, need wrapper component? :O
This will be even more things to look after now..

Reply:
I mean.. Why we cannot just use `TheemProvider` straight in the component, and need to create a wrapper component instead?

Reply:
`useToaster` is a hook that use a context that is defined in `ThemeProvider`, to use that in the component before the `return`, the hook need access to the `React.context`, so it need to be further up the chain.

Reply:
oh.. understood..